doc/API.md

```diff
@@ -443,24 +443,26 @@ The Datadog nginx module defines additional variables that provide information
 about the currently active trace.
 
 ### `datadog_trace_id`
-`$datadog_trace_id` expands to the decimal representation of the unsigned
-64-bit ID of the currently active trace. If there is no currently active
+`$datadog_trace_id` expands to the hexadecimal representation of the unsigned
+128-bit ID of the currently active trace. If there is no currently active
 trace, then the variable expands to a hyphen character (`-`) instead.
 
 ### `datadog_span_id`
-`$datadog_span_id` expands to the decimal representation of the unsigned 64-bit
+`$datadog_span_id` expands to the hexadecimal representation of the unsigned 128-bit
 ID of the currently active span.If there is no currently active span, then
 the variable expands to a hyphen character (`-`) instead.
 
 Note that if `datadog_trace_locations` is `on`, then `$datadog_span_id` will
 refer to the span associated with the location (outbound request), not its
 parent (inbound request).
 
-### `datadog_trace_id_hex`
-Same as `$datadog_trace_id`, but is the hexadecimal representation of the 128-bit ID. 
+### `datadog_trace_id_dec`  
```

Comment:
Since naming is a fun engineering problem 😄 WDYT about `_base10` instead of `_dec`? Not blocking, just a suggestion

Reply:
Why not. how about `_short_base10`.

Reply:
Sounds good to me!